src/dependencies.scala

```diff
@@ -150,5 +173,52 @@ object Dependencies {
         deps map { x => x: ClasspathDep[ProjectReference] }:_*)
     }
   }
+
+  implicit class ModuleIDOps(id: ModuleID)
+  {
+    def revMatches(other: String) = {
+      def partMatches(p: (String, String)) = p._1 == p._2 || p._1 == "+"
+      val parts = id.revision.toString.split('.')
+      val otherParts = other.toString.split('.')
+      val partsMatch = parts zip(otherParts) forall(partMatches)
+      partsMatch && (
+        parts.length == otherParts.length ||
+        (parts.length < otherParts.length) && parts.lastOption.exists(_ == "+")
+      )
+    }
+
+    def matches(other: ModuleID) = {
+      id.organization == other.organization && id.name == other.name &&
+        revMatches(other.revision)
+    }
+
+    def isAar = id.explicitArtifacts.exists(_.`type` == "aar")
+  }
+
+  implicit class ProjectRefOps(project: ProjectRef)
+  (implicit struct: sbt.BuildStructure)
+  {
+    def resolved = Project.getProject(project, struct)
+
+    def deps = resolved map(_.dependencies) getOrElse(Nil) map(_.project)
+
+    def deepDeps: Seq[ProjectRef] =
+      ((deps flatMap(_.deepDeps)) :+ project).distinct
+
+    def libraryDependencies =
+      (sbt.Keys.libraryDependencies in project)
+        .get(struct.data)
+        .getOrElse(Nil)
+
+    def aarDependencies = libraryDependencies filter(_.isAar)
```

Comment:
get rid of this and associated deepAarDeps since it doesn't work in this situation